### PR TITLE
Lw 244 add new hours text

### PR DESCRIPTION
**Pull request by jonhartzler**
I updated the display of the hours to catch if there is an api failure.   

I added a component to show the error message.  It looks to see if there is a service point and if that service point has a phone number and shows that service point's data instead of hesburgh library.  

IMPORTANT:
I updated the APIPresenter to take a new parameter that is an error object to display if it is determining the the api's are in error.  This is so the /hours page does not show a 500 error message.

### CODE REVIEW

File: src/actions/hours.js [hunk 1]
```diff
@@ -13,11 +13,12 @@ export const requestHours = () => {
 }
 
 const receiveHours = (response) => {
-  if (response === 'TypeError: Failed to fetch') {
+  if (response == 'TypeError: Failed to fetch') {
```
[the remark this answers is not in the document]
Comment (jonhartzler) [reply src/actions/hours.js]: === matches as false so i set == and it works.

File: src/components/Hours/HomePage/index.js [hunk 2]
```diff
@@ -43,6 +43,7 @@ export class HomePageHoursContainer extends Component {
   }
 
   render () {
+    console.log(this.props.hoursEntry.status)
```
Comment (hbeachey) [inline src/components/Hours/HomePage/index.js]: Remove debug

File: src/actions/hours.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (danthewolfe) [reply src/actions/hours.js]: Does `new String(response) === 'TypeError: Failed to fetch'` work?

[the remark this answers is not in the document]
Comment (jonhartzler) [reply src/actions/hours.js]: yes it does not match.

File: src/components/Hours/Error/index.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+const Error = (props) => {
+  let title = 'circulation desk'
+  let phone = '(574) 631-6679'
+  if (servicePointExists(props)) {
+    title = props.hoursEntry.servicePoint.title
+    phone = props.hoursEntry.servicePoint.phoneNumber
+  }
+  return (
+    <section className='hours-display'><h2>The hours are currently unavailable. <br />Please call the {title} at {phone}.</h2></section>
```
[the remark this answers is not in the document]
Comment (danthewolfe) [reply src/components/Hours/Error/index.js]: Please split up into separate lines.

**Pull request merged**